packages/relay/src/lib/eth.ts

```diff
@@ -558,6 +559,8 @@ export class EthImpl implements Eth {
       `${requestIdPrefix} estimateGas(transaction=${JSON.stringify(transaction)}, _blockParam=${_blockParam})`,
     );
 
+    formatTransaction(transaction);
```

Comment:
Can you add an `eth.spec.ts` where you intentionally set the `transaction.input` instead of `transaction.data`

Reply:
Added.